File: Maximum Number/maximumnumber.java
import java.util.Scanner;

public class maximumnumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a, highest = 0;

        while (scanner.hasNextInt() && (a = scanner.nextInt()) >= 0) {
            if (a > highest) {
                highest = a;
            }
        }

        System.out.printf("%d", highest);
    }
}

File: Maximum Number/maximumnumber.c
#include <stdio.h>

int main() {
    int a, highest = 0;

    while (scanf("%d", &a) == 1 && a >= 0) {
        if (a > highest) {
            highest = a;
        }
    }

    printf("%d", highest);
    return 0;
}
